jsk_recognition_msgs/msg/Label.msg

```diff
@@ -0,0 +1,2 @@
+int32 label
```

Comment:
I thought about for a while last night, and maybe `id` is a better name. What is your idea?